guardian/formstack-submitter-ts
# Bump node and dependencies to v14

## What does this change?
Upgrade to Node 14 (that's the latest supported by Lambda according to [this page](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html))

While we're here we're also bumping some dependencies.

## How to test
Adjust the build in TeamCity to use the CI script instead of using docker to install node. If the build succeeds, update the cloudformation template on PROD, and deploy this branch. We can't test the CODE version of this lambda, as the CODE environment for Formstack callouts is using the PROD version of this lambda.

To trigger the lambda, and test if working as normal:
(1) find tag for live formstack form in https://targeting.gutools.co.uk/
(2) find article with that tag
(3) submit dummy data to form (in PROD)
(4) check lambda dashboard for data

____

UPDATE: The above has been tested successfully.

## How can we measure success?
If it works, success!

## Have we considered potential risks?
If it breaks we'll have to revert the cloudformation and redeploy the previous known good build while we rethink.

## Images
N/A

## Accessibility
N/A
